### DISCUSSION TITLE
人工智慧淪為寫作業神器，紐約公立學校禁用 ChatGPT

### DISCUSSION THREAD

**Discussion by neil0258** [1101-GS3073-C/Python-Community-GS3073-C#627] [General]
ChatGPT 上線後聲名大噪，對許多領域的未來帶來全新想像，也可能造成衝擊，尤其教育領域有正反討論，一方認為使用 ChatGPT 是作弊、抄襲，而且產出的文字內容常常似是而非，另一方則認為 ChatGPT 具有教學潛力，為師生帶來不同體驗。
國外媒體 TechCrunch 詢問 OpenAI 看法，回應正在開發對應措施，幫助人們得知文字內容是否由 ChatGPT 產生
另一派教育工作者持不同看法，看出 ChatGPT 的教學潛力。比方說，史丹佛大學教育學副教授 Victor Lee 發表專文指出，ChatGPT 可以幫助學生「以目前沒有想過的方式思考」，幫助他們發現和闡明不同想法。他也假設教師能在基礎內容不變的條件下，為學生提供不同風格、句法、語法的敘述示範。他認為 ChatGPT 可讓學生多次閱讀、反思並修改，不像傳統學習歷程常常感到沮喪，教師也能使用這項工具產生更多範例。
教育界對 ChatGPT 的正反討論不會停止，雖然目前在探索階段提出疑慮，不代表就此封殺，仍會持續找出適合師生學習的有效做法。

我認為：AI 帶來了利與弊，雖然chatgpt或許會傷害了我們自主思考、批判的能力，且對仍在發育學習的青少年更是如此；然而，如果我們能從ChatGPT中獲取我們之前從沒想到的方向，並以此為出發點，提出更不一樣的思維感受，或許對社會進步有更顯著的幫助

https://technews.tw/2023/01/06/nyc-education-department-blocks-chatgpt-on-school-devices-and-networks/

**Comment by chia-kai-chang** [comment]
根據[最新的討論區規則](https://github.com/NCU-GS3073-AI-Introduction/ai-introduction-community/wiki/1111-%E3%80%8CAI%E4%BA%BA%E5%B7%A5%E6%99%BA%E6%85%A7%E5%B0%8E%E8%AB%96%E3%80%8D%E8%A8%8E%E8%AB%96%E5%8D%80%E7%A9%8D%E5%88%86%E8%A8%88%E7%AE%97%E8%A6%8F%E5%89%87)

> 三種貼文若被教師或助教用Emoji 😄 標示，該貼文排除AI判斷結果

因此你的貼文被老師守護~

**Comment by chia-kai-chang** [comment]
大數學家陶哲軒對於風起雲湧的AI的看法，甚為合理。未來的AI工具就像是工程計算機一樣，學生要學習正確使用AI工具。
我不反對大家使用AI，但不能亂用，要用的恰到好初，讓AI真正幫助我們產出好的文章或作品阿
(唬爛產生器請不要來亂)

![image](https://user-images.githubusercontent.com/1768891/211015089-78a758b4-72bc-4f2b-8f3b-f05961a93dff.png)

**Comment by magicmicheal** [comment]
欸不是我夢到其他通識課，音樂素養之類的聽一首歡樂頌，一個小時的音樂會，然後要打四面A4多的新的，我這種沒甚麼文化的人在這份作業上肯定很需要這項功能的幫助阿

**Comment by boson13579** [comment]
其實從另一個方面想，ChatGPT也是有好處的。他可以教你從不同方面看同一種資訊，而且也會幫你彙整出網路上相關的資訊。所以適當並合理的運用AI為我們提供的訊息才是解決之法。
